src/Provider/Doctrine/Auditing/Transaction/TransactionHydrator.php

```diff
@@ -124,11 +125,12 @@ private function hydrateWithScheduledCollectionDeletions(Transaction $transactio
                 /** @var object $entity */
                 foreach ($collection->toArray() as $entity) {
                     if ($this->provider->isAudited($entity)) {
-                        $transaction->trackAuditEvent(Transaction::DISSOCIATE, [
+                        $transaction->dissociate(
                             $collection->getOwner(),
                             $entity,
+                            $this->id($entityManager, $entity),
```

Comment:
I'm not quite sure why the ID is part of the dissociation event, however, the test suite on the latest master was failing, and adding the ID to the event object fixed the problem. I smell dead code that exists just because there is a test for? 

```
Runtime:       PHP 8.1.4 with PCOV 1.0.11
Configuration: /Users/janklan/dev/auditor/phpunit.xml.dist

..........E....................................................  63 / 142 ( 44%)
............................................................... 126 / 142 ( 88%)
................                                                142 / 142 (100%)

Time: 00:03.774, Memory: 48.00 MB

There was 1 error:

1) DH\Auditor\Tests\Provider\Doctrine\Auditing\Transaction\TransactionProcessorTest::testProcessDissociations
Undefined array key 3

/Users/janklan/dev/auditor/src/Provider/Doctrine/Auditing/Transaction/TransactionProcessor.php:162
/Users/janklan/dev/auditor/src/Provider/Doctrine/Auditing/Transaction/TransactionProcessor.php:41
/Users/janklan/dev/auditor/tests/Provider/Doctrine/Auditing/Transaction/TransactionProcessorTest.php:770
```

Reply:
The ID should not be part of the dissociation. I recently removed it (#93)

Reply:
> @janklan Thanks for this nice contribution!

Happy to help. It's a good library, thanks for the load of work you do maintaining it.